go/multipooler/manager/rpc_manager.go

```diff
@@ -250,6 +250,71 @@ func (pm *MultiPoolerManager) ReplicationStatus(ctx context.Context) (*multipool
 	return status, nil
 }
 
+// ReplicationStatus gets unified status that works for both PRIMARY and REPLICA poolers
+// The multipooler returns information based on what type it believes itself to be
```

Comment:
It seems fine to call this `ReplicationStatus` right now because `PrimaryStatus` pretty much returns only replication related info. Are we sure that in future we won't be adding more things to it that are unrelated to replication?
One alternative is to call this `Status` and rename the existing `Status` RPC to `State` - because that is what it is returning.